.github/workflows/sycl-benchmark.yml

```diff
@@ -0,0 +1,215 @@
+name: Run benchmarks (using sycl-bench)
+
+on:
+  workflow_call:
+    inputs:
+      runner:
+        type: string
+        required: True
+      image:
+        type: string
+        required: True
+      image_options:
+        type: string
+        required: True
+
+      sycl_toolchain_artifact:
+        type: string
+        default: ''
+        required: False
+      sycl_toolchain_archive:
+        type: string
+        default: ''
+        required: False
+      sycl_toolchain_decompress_command:
+        type: string
+        default: ''
+        required: False
+
+  workflow_dispatch:
+    inputs:
+      runner:
+        type: choice
+        options:
+          - '["Linux", "gen12"]'
+          - '["amdgpu"]'
+          - '["Linux", "arc"]'
+          - '["cts-cpu"]'
+      image:
+        description: |
+          Use option ending with ":build" for AMDGPU, ":latest" for the rest.
+        type: choice
+        options:
+          - 'ghcr.io/intel/llvm/sycl_ubuntu2204_nightly:latest'
+          - 'ghcr.io/intel/llvm/sycl_ubuntu2204_nightly:build'
+      image_options:
+        description: |
+          Use option with "--device=/dev/kfd" for AMDGPU, without it for the rest.
+        type: choice
+        options:
+          - '-u 1001 --device=/dev/dri --privileged --cap-add SYS_ADMIN'
+          - '-u 1001 --device=/dev/dri --device=/dev/kfd --privileged --cap-add SYS_ADMIN'
+
+      sycl_toolchain_artifact:
+        type: string
+        default: ''
+        required: False
+      sycl_toolchain_archive:
+        type: string
+        default: ''
+        required: False
+      sycl_toolchain_decompress_command:
+        type: string
+        default: ''
+        required: False
+
+
+permissions:
+  contents: read
+
+jobs:
+  run:
+    name: Run benchmarks using sycl-bench
+    runs-on: ${{ fromJSON(inputs.runner || '["Linux", "gen12"]') }}
+    container:
+      image: ${{ inputs.image }}
+      options: ${{ inputs.image_options }}
+    steps:
+    - name: Download SYCL toolchain
+      if: inputs.sycl_toolchain_artifact != '' && github.event_name != 'workflow_run'
+      uses: actions/download-artifact@v4
+      with:
+        name: ${{ inputs.sycl_toolchain_artifact }}
+    - name: Debug prints [workflow_run]
+      if: inputs.sycl_toolchain_artifact != '' && github.event_name == 'workflow_run'
+      run: |
+        pwd
```

Comment:
I think we can get rid of `pwd` and `ls`

Reply:
Fixed in #14454